src/cards/top-languages-card.js

```diff
@@ -203,20 +204,34 @@ const trimTopLanguages = (topLangs, langs_count, hide) => {
  * @param {number} props.width The card width
  * @param {string} props.color Color of the programming language.
  * @param {string} props.name Name of the programming language.
- * @param {number} props.progress Usage of the programming language in percentage.
+ * @param {number} props.size Size of the programming language.
+ * @param {number} props.totalSize Total size of all languages.
+ * @param {string} props.statsFormat Stats format.
  * @param {number} props.index Index of the programming language.
  * @returns {string} Programming language SVG node.
  */
-const createProgressTextNode = ({ width, color, name, progress, index }) => {
+const createProgressTextNode = ({
+  width,
+  color,
+  name,
+  size,
+  totalSize,
+  statsFormat,
+  index,
+}) => {
   const staggerDelay = (index + 3) * 150;
   const paddingRight = 95;
   const progressTextX = width - paddingRight + 10;
   const progressWidth = width - paddingRight;
 
+  const progress = parseFloat(((size / totalSize) * 100).toFixed(2));
+  const bytes = formatBytes(size);
+  const showValue = statsFormat === "bytes" ? bytes : `${progress}%`;
```

Comment:
I see that this file contains the almost same code on 266-269 lines. I think it will be better to avoid duplication of this logic and create separate function.

Reply:
I'd prefer it that way too, modified at https://github.com/anuraghazra/github-readme-stats/pull/3708/commits/0c90270143bdb0aaeab32d9887cbed8b4d0953b7.